hw/dv/tools/dvsim/sim.mk

```diff
@@ -88,8 +88,27 @@ ifneq (${sw_images},)
 			cp ${proj_root}/$$image* $$target_dir/.; \
 		else \
 			echo "Building SW image \"$$image\"."; \
-			target="$$image""_export_${sw_build_device}"; \
-			${LOCK_SW_BUILD_DIR} "ninja -C ${sw_build_dir}/build-out $$target"; \
+			if [[ $$index == "0" ]]; then \
+				target="$$image""_${sw_build_device}"; \
+				if [[ -z $${BAZEL_PYTHON_WHEELS_REPO} ]]; then \
+					echo "Building \"//$$(dirname $$image)\" with bazel on network-connected machine."; \
+					./bazelisk.sh build //$$(dirname $$image); \
+				else \
+					echo "Building \"//$$(dirname $$image)\" with bazel on air-gapped machine."; \
+					bazel build \
+						--distdir=$$BAZEL_DISTDIR \
+						--repository_cache=$$BAZEL_CACHE \
+						//$$(dirname $$image); \
+				fi; \
+				image_basename="$$(basename $${target}).scr.39.vmem"; \
+				bazel_image_path=$$(find -L $$(bazel info output_path)/ -type f -name "$${image_basename}"); \
```

Comment:
Above you use `bazelisk.sh` on a network-connected machine, but here you're using `bazel`.  If the environment doesn't know where `bazel` is above, how does it know here?

Reply:
On the air-gapped machine, it is required that the appropriate bazel binary is pre-installed on the machine, and its location on the system is added to the `PATH` environment variable. This is detailed in my "Bazel Air-gapped Builds" document.

Reply:
This is failing (private) CI do to a different reason. In fact, the private CI machines are connected to the network, so they make use of the `bazelisk.sh` script. The private CI tests that are failing here, are actually passing when run on the air-gapped nightly regression machine.

Reply:
Oh sorry, I just realized what you were referring to, thats a good catch! This actually may impact why this is failing, private CI. Let me fix this and see if it helps.

Reply:
That did the trick. Thanks @cfrantz !